Funcky.Money/Exceptions/MissingExchangeRateException.cs

```diff
@@ -0,0 +1,26 @@
+using System.Runtime.Serialization;
+
+namespace Funcky;
+
+public class MissingExchangeRateException : Exception
+{
+    public MissingExchangeRateException()
+        : base("if you calculate with more than one currency, you have to define the exchange rate and target in the evaluation context.")
```

Comment:
We usually don't use trailing dots and we capitalize the first letter

Reply:
Fixed capitalization: 84e7e9a8cb162e30712b22243d434bb33bff83fc. A sentence needs a full stop,

examples: 
* NullReferenceException -> Object reference not set to an instance of an object.
* DivideByZeroException -> Attempted to divide by zero.

Reply:
See my other comment and resolve this yourself if you want!